### ISSUE TITLE
MASTAnalysis.Rmd is not reproducible

### ISSUE THREAD

**Issue by gokceneraslan** [RGLab/MASTdata#1]
Could you please include the `rds` file that is being referred to in the following line to the package:

```
gsea_exclusive_other_methods<-readRDS("~/Downloads/significant_genes_q001_fc13.rds")
```

**Comment by gfinak** [comment]
Will do, thanks for pointing this out. 
I'll push a fix shortly.

**Comment by gfinak** [comment]
fixed in 128f6b8

**Comment by gokceneraslan** [comment]
Thanks!
